### ISSUE TITLE
OneStatementPerLine check and lambda expression

### ISSUE THREAD

**Issue by richardSch** [checkstyle/checkstyle#2688]
The OneStatementPerLine flags the lambda expression as containing multiple statements.  I want it to not report an error in this case.  If there were multiple statements inside the lambda on the same line, then I would expect an error.

http://checkstyle.sourceforge.net/config_coding.html#OneStatementPerLine

% java -jar checkstyle-6.13-all.jar -c checks.xml Y.java
Starting audit...
/run/shm/Y.java:3:80: error: Only one statement per line allowed.
Audit done.
Checkstyle ends with 1 errors.
% cat Y.java
```java
public class Y {
    void y() {
        new JCheckBox().addActionListener((final ActionEvent e) -> { good(); });
    }
}
```
% cat checks.xml 
```xml
<?xml version="1.0"?>
<!DOCTYPE module PUBLIC "-//Puppy Crawl//DTD Check Configuration 1.2//EN" "http://www.puppycrawl.com/dtds/configuration_1_2.dtd">
<module name="Checker">
    <property name="charset" value="UTF-8"/>
    <module name="TreeWalker">
        <module name="OneStatementPerLine"/>
    </module>
</module>
```

**Comment by romani** [comment]
please read http://checkstyle.sourceforge.net/report_issue.html#How_to_report_a_bug

**Comment by richardSch** [comment]
The OneStatementPerLine flags the lambda expression as containing multiple statements.  I want it to not report an error in this case.  If there were multiple statements inside the lambda on the same line, then I would expect an error.

% java -jar checkstyle-6.13-all.jar -c checks.xml Y.java
Starting audit...
/run/shm/Y.java:3:80: error: Only one statement per line allowed.
Audit done.
Checkstyle ends with 1 errors.

% cat Y.java
public class Y {
    void y() {
        new JCheckBox().addActionListener((final ActionEvent e) -> { good(); });
    }
}

% cat checks.xml 
<?xml version="1.0"?>
<!DOCTYPE module PUBLIC "-//Puppy Crawl//DTD Check Configuration 1.2//EN" "http://www.puppycrawl.com/dtds/configuration_1_2.dtd">
<module name="Checker">
    <property name="charset" value="UTF-8"/>
    <module name="TreeWalker">
        <module name="OneStatementPerLine"/>
    </module>
</module>

**Comment by richardSch** [comment]
Sorry, my previous message cut off the checks.xml file:

% cat checks.xml 
<?xml version="1.0"?>
<!DOCTYPE module PUBLIC "-//Puppy Crawl//DTD Check Configuration 1.2//EN" "http://www.puppycrawl.com/dtds/configuration_1_2.dtd">
<module name="Checker">
    <property name="charset" value="UTF-8"/>
    <module name="TreeWalker">
        <module name="OneStatementPerLine"/>
    </module>
</module>

**Comment by romani** [comment]
thanks a lot!

**Comment by romani** [comment]
fix is merged.
